engine/vector/src/main/java/io/deephaven/vector/CharVector.java

```diff
@@ -99,6 +100,21 @@ default String toString(final int prefixLength) {
         return toString(this, prefixLength);
     }
 
+    /**
+     * <p>
+     * Compare this vector with another vector.
+     * </p>
+     *
+     * <p>
+     * The vectors are ordered lexicographically, producing an order consistent with
+     * {@link Arrays#compare(char[], char[])}.
+     * </p>
+     *
+     * {@inheritDoc}
```

Comment:
I think this might make the JavaDoc worse, not better. Consider `     * @see Comparable#compareTo(Object) `